src/libs/ReportUtils.ts

```diff
@@ -1327,7 +1327,7 @@ function isSettled(reportOrID: OnyxInputOrEntry<Report> | SearchReport | string
     // In case the payment is scheduled and we are waiting for the payee to set up their wallet,
     // consider the report as paid as well.
     if (report.isWaitingOnBankAccount && report.statusNum === CONST.REPORT.STATUS_NUM.APPROVED) {
-        return true;
+        return false;
     }
```

Comment:
I am a bit worried about change like this - first the comment is wrong now and could you please highlight the RCA that caused this issue?